be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -236,6 +692,26 @@ RowsetSharedPtr BetaRowsetWriter::build() {
             return nullptr;
         }
     }
+    Status status;
```

Comment:
Not related to this pr, we should close file_writer timely, otherwise we consumes a lot fds.

Reply:
sure.